Configure.old

```diff
@@ -0,0 +1,3535 @@
+#! /usr/bin/env perl
+# -*- mode: perl; -*-
+# Copyright 2016-2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
I think you've added this file by accident.

Reply:
removed, thanks